### ISSUE TITLE
Update people names in post list to strong tag instead of an h3 tag

### ISSUE THREAD

**Issue by rjucf** [UCF/UCF-WordPress-Theme#65]
**Describe the bug**

The people names in the people list should be changed to a strong tag instead of an H3 tag here:
https://github.com/UCF/UCF-WordPress-Theme/blob/master/includes/post-list-functions.php#L100

**Steps to Reproduce**

An example can be found here:
https://www.ucf.edu/admissions/undergraduate/contact/

**Expected Behavior**

strong tag instead of h3 tag

**Comment by jmbarne3** [comment]
Resolvedin #80
